addons/ofxOpenCv/scripts/formulas/opencv.sh

```diff
@@ -494,10 +582,10 @@ function build() {
 function copy() {
 
   # prepare headers directory if needed
-  mkdir -p $1/include
+  mkdir -p $1/../../addons/ofxOpenCv/libs/opencv/include
```

Comment:
It was looking in the wrong path for me. It was looking in the libraries folder instead of the addons path it should have been directed too.

Dominic Amato
Human Computer Interaction Masters Candidate
Physical Computing Lab (http://goo.gl/dguGzt)
DePaul University

-----Original Message-----
From: "arturo" <notifications@github.com>
Sent: ‎7/‎10/‎2015 11:48 AM
To: "openframeworks/openFrameworks" <openFrameworks@noreply.github.com>
Cc: "Dominic Amato" <dominic.a.amato@gmail.com>
Subject: Re: [openFrameworks] VS OpenCV lib and script Updates (#4086)

In addons/ofxOpenCv/scripts/formulas/opencv.sh:
> @@ -494,10 +582,10 @@ function build() {
>  function copy() {
>  
>    # prepare headers directory if needed
> -  mkdir -p $1/include
> +  mkdir -p $1/../../addons/ofxOpenCv/libs/opencv/include
why this? it was working in other platforms as it was. perhaps we need to fix the path passed to this function in windows?
—
Reply to this email directly or view it on GitHub.